### PR TITLE
Integra

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/integra/aquastream/alarm.schema.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "$id": "https://akenza.io/integra/aquastream/alarm.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Alarm",
+  "type": "object",
+  "properties": {
+    "tamper": {
```
Comment (mariusgiger) [inline types/integra/aquastream/alarm.schema.json:7]: make the data keys speak for themselves, e.g. tamperAlarmTriggered

File: types/integra/aquastream/default.schema.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://akenza.io/integra/aquastream/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "volume": {
+      "title": "Volume",
+      "unit": "L",
```
Comment (mariusgiger) [inline types/integra/aquastream/default.schema.json:9]: l

File: types/integra/aquastream/default.schema.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      "type": "number",
+      "description": "Total volume in liter"
+    },
+    "volumeM3": {
```
Comment (mariusgiger) [inline types/integra/aquastream/default.schema.json:13]: is there a description when cubic meters vs liters are used? if yes try to rename the data key according to this description

File: types/integra/aquastream/default.schema.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      "title": "Volume M3",
+      "unit": "m^3",
+      "type": "number",
+      "description": "Total volume in cubic meters"
+    },
+    "backflow": {
+      "title": "Backflow",
+      "unit": "L",
```
Comment (mariusgiger) [inline types/integra/aquastream/default.schema.json:21]: l

File: types/integra/aquastream/device.schema.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    "meterVersion": {
+      "title": "Meter version",
+      "type": "integer",
+      "description": "Meter version",
+      "hideFromKpis": true
+    },
+    "waterType": {
+      "title": "Water type",
+      "type": "integer",
+      "description": "Either warm or cold",
```
Comment (mariusgiger) [inline types/integra/aquastream/device.schema.json:46]: use an ENUM? or describe in which case it is 0 and 1

File: types/integra/aquastream/meta.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Aquastream",
+  "version": "1.0.0",
+  "manufacturer": "Integra",
+  "url": "https://ch.integra-metering.com/product/aquastream-m-bus/",
+  "description": "System Modul Mod-Bus for mechanical water metering",
```
Comment (mariusgiger) [inline types/integra/aquastream/meta.json:6]: Type Module?

File: types/integra/aquastream/device.schema.json [hunk 7]
```diff
@@ -0,0 +1,74 @@
+{
+  "$id": "https://akenza.io/integra/aquastream/device.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Device",
```
Comment (mariusgiger) [inline types/integra/aquastream/device.schema.json:4]: why is this not called lifecycle as always?

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, a few small comments

File: types/integra/aquastream/device.schema.json (hunk 7 above)
Comment (J2thatsme) [reply types/integra/aquastream/device.schema.json:4]: renamed it

File: types/integra/aquastream/device.schema.json (hunk 5 above)
Comment (J2thatsme) [reply types/integra/aquastream/device.schema.json:46]: Theres sadly no description to this datapoint.

**Review by sevyakenza — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]
looks good

**Pull request merged**